src/angleaxis_types.jl

```diff
@@ -44,14 +44,11 @@ end
 end
 
 # These functions are enough to satisfy the entire StaticArrays interface:
-@inline (::Type{AA})(t::NTuple{9}) where {AA <: AngleAxis} = convert(AA, Quat(t))
-@inline Base.getindex(aa::AngleAxis, i::Int) = convert(Quat, aa)[i]
-@inline Tuple(aa::AngleAxis) = Tuple(convert(RotMatrix, aa))
+@inline (::Type{AA})(t::NTuple{9}) where {AA <: AngleAxis} = AA(Quat(t)) # TODO: consider going directly from tuple (RotMatrix) to AngleAxis
+@inline Base.getindex(aa::AngleAxis, i::Int) = Quat(aa)[i]
 
-@inline function Base.convert(::Type{R}, aa::AngleAxis) where R <: RotMatrix
+@inline function Base.Tuple(aa::AngleAxis{T}) where T
```

Comment:
Change to a `Base.Tuple` constructor to match existing `Quat` code. The constructors in StaticArrays' `convert.jl` make this work.